src/appkit.cs

```diff
@@ -25789,9 +25789,7 @@ interface NSAccessibilityCustomAction
 		[NullAllowed, Export ("target", ArgumentSemantic.Weak)]
 		NSObject Target { get; set; }
 
-		[Advice (@"It must conform to one of the following signatures: 
-		- (BOOL)myPerformActionMethod;
-		- (BOOL)myPerformActionMethod:(NSAccessibilityCustomAction *)action;")]
+		[Advice (@"It must conform to one of the following signatures: bool ActionMethod () or bool ActionMethod (NSAccessibilityCustomAction action) and be decorated with a corresponding [Export]")]
```

Comment:
Please add apostrophes and dot at the end (:

`It must conform to one of the following signatures: 'bool ActionMethod ()' or 'bool ActionMethod (NSAccessibilityCustomAction action)' and be decorated with a corresponding '[Export]'.`